src/main.rs

```diff
@@ -21,7 +21,8 @@ fn main() -> Result<(), Box<dyn Error>> {
         if terms != expected {
             panic!("terms.toml is not sorted");
         }
-        // more checks to be added
+        // check #2: make sure all tags are valid
+        Terms::check_all_tags(&terms.terms)?;
```

Comment:
not sure about what's a better pattern here: `terms.check_all_tags` or something like this 😛